wiki/Beatmapping/Hitsound/en.md

```diff
@@ -21,13 +21,15 @@ Beatmappers can replace any default sound samples with custom samples using a [b
 
 ## Active hitsound
 
-An active hitsound correlates to a player's clicking by reaching its peak impact immediately when played. Drum sound samples are the most frequently used active hitsounds because they provide clear and immediate feedback.
+A hitsound is considered *active* if it is on a clickable hitobject edge, for example a circle or slider head. These help the player judge whether they clicked early or late by producing a clear impact sound at the time of clicking. If clicked correctly, the sound should match up with the beats in the song the objects are mapped to.
 
 ## Passive hitsound
 
-A passive hitsound does not correlate to a player's clicking. These are usually sound effects that accompany active hitsounds.
+If a hitsound is on a non-clickable edge, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hitobject, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.
 
-Hitsounds can be added through storyboarding, but because they don't correlate to a user's clicking, they are also classified as passive.
+## Storyboarded hitsound
+
+Sound samples may be added through storyboarding to mimic hitsounds on objects. Unlike actual hitsounds, however, these always play at the given time regardless of whether the player hit the object. For that reason, they are often not recommended outside of osu!mania.
```

Comment:
Why is mania a special case here?

Reply:
can probably split off the part abt mania into a second small paragraph with an explanation of why it's only recommended there (i dont know why myself but assuming it's relevant). otherwise it feels pretty random to include here

Reply:
Storyboarded hitsounds are way more common in mania than osu (from observation), and i think it's to counteract many hitsounds playing in short succession due to players not hitting multiple notes quite simultanously or quick stairs quite in time, etc (can sound bad on fast maps even if you're holding an SS)

Reply:
Yeah I'm not sure. In mania you could just attach hitsounds to one column, that's usually the standard way. I'd probably drop the mania part.

Reply:
Mainly based this off of

![image](https://user-images.githubusercontent.com/30292137/149966685-d4ee73a4-0605-4116-9d65-819e01c23944.png)

but after asking a few mappers/modders from mania, it turns out they more or less just use them as a last resort when regular hitsounding is either not possible (e.g. to fill the gap between notes) or would require more than one object (e.g. for complex hitsounding or keysounding where multiple samples are used); probably for the reason Walavouchey described.

Either way, dropped the mania part.